build/ci/evergreen.yml

```diff
@@ -861,49 +870,6 @@ tasks:
       - func: "e2e test"
         vars:
           E2E_TAGS: opsmanager,remote,sharded
-  # Deploy a replica set, we need to be careful on running parallel modifications to the automation config
```

Comment:
not going to happen given how close we are to EOL

Reply:
[q] 4.2 tests are failing, is there a plan for a 4.4 version of the same test or it's no longer necessary?

Reply:
we never enabled them as there was an automaton test failing due to changes between 4.2 and 4.4
We already have OM4.4 tests in place TBD 5.0 tests but we have cloud manager tests which at this point they are almost the same

Reply:
also mongocli was always a 4.4 first 4.2 best effort (given timings between the launch of mcli and 4.4(

Reply:
Cool, thanks for the context!